common/controllers/framework/confirm-assessment.js

```diff
@@ -17,6 +19,28 @@ class ConfirmAssessmentController extends FormWizardController {
     res.redirect(`/move/${moveId}`)
   }
 
+  async saveValues(req, res, next) {
+    const { assessment } = req
+    const services = {
+      'person-escort-record': req.services.personEscortRecord,
+      'youth-risk-assessment': req.services.youthRiskAssessment,
+    }
+
+    try {
+      await services[assessment.framework.name].confirm(assessment.id)
+
+      next()
+    } catch (err) {
+      const apiErrorCode = get(err, 'errors[0].code')
+
+      if (err.statusCode === 422 && apiErrorCode === 'taken') {
```

Comment:
This is blocked until we know the code that the API will return so that we can catch it.

cc @cwrw